<ui version="4.0" >
 <class>DlgBpmTapDlg</class>
 <widget class="QDialog" name="DlgBpmTapDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BPM Editor</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <widget class="Q3GroupBox" name="grpBoxDetection" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>350</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>BPM Detection</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>63</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>BPM Scheme:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboSchemes" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>60</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>90</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>to</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBPMRangeStart" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>90</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>220</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBPMRangeEnd" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>220</number>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>74</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>BPM Range:</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBPMDetect" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>28</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="textVisible" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGo" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>28</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Go</string>
    </property>
    <property name="shortcut" >
     <string>Alt+G</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>230</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Close</string>
   </property>
   <property name="shortcut" >
    <string>Alt+O</string>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblSong" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text" >
    <string>Song:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>230</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Next >></string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrev" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;&lt; Prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTap" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>127</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Push to tap tempo</string>
   </property>
   <property name="shortcut" >
    <string>Alt+P</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
   <property name="default" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTempo" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>183</y>
     <width>30</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>BPM:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBPM" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>180</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
